src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -225,6 +233,17 @@ public unsafe static bool TryGetPublicKey(ImmutableArray<byte> blob, out Immutab
                 // The RSA magic key must match the blob id
                 if (type == PrivateKeyBlobId && magic == RSA2)
                 {
+                    RSAParameters privKey = default;
```

Comment:
Hmm.  No idea.  Seems like it would be pretty easy to solve though.

`expBytes.DropWhile(b => b == 0).ToArray()`